doc/lua_api.txt

```diff
@@ -6297,9 +6299,13 @@ Misc.
     * `method` is a string identifying the compression method to be used.
     * Supported compression methods:
         * Deflate (zlib): `"deflate"`
+        * Zstandard: `"zstd"`
     * `...` indicates method-specific arguments. Currently defined arguments
       are:
         * Deflate: `level` - Compression level, `0`-`9` or `nil`.
+        * Zstandard: `level` - Compression level. `0`-`3` or `nil`.
```

Comment:
Why 0-3? In-engine I mapped useful zstd compression levels to deflates 0-9 so that we have the same scale.

Reply:
See earlier review comments, it looks like the library only guarantees 0-3 to exist (I assume with some very exotic compile options).
IMO better to not attempt to document it here in that case.

Reply:
> Outright remove the level from being documented, or...?

yes, e.g.
<pre>
* Zstandard: `level` - Compression level. positive integer or `nil`
</pre>

Reply:
Adjusted in c50c720 but with some specifics to the wording as mentioned.